src/pages/docs/sdk/getting-started/considerations.md

```diff
@@ -50,3 +50,8 @@ If you need to adapt the *look and feel*, each component of the card has been as
 * **microlink_card__content_title**: The `p` tag of the card title.
 * **microlink_card__content_description**: The `p` tag of the card description.
 * **microlink_card__content_url**: The `span` tag of the card url link.
+
+For the [Vanilla](/docs/sdk/integrations/vanilla) and [Vue](/docs/sdk/integrations/vue) packages there are some specific class names:
+
+* **microlink_vanilla_dom**: A root div used to render the React DOM.
+* **microlink_vue_dom**: A root div used to render the React DOM.
```

Comment:
Maybe we should say something instead of "React DOM" @Kikobeats, might be confusing to a user who doesn't fully understand that React is being used under the hood. We could replace with:

* ...used to render the virtual DOM.
* ...used to wrap the card.
* ...?

Reply:
`virtual DOM` or `node root` sound good to me 🙂